synapseclient/__main__.py

```diff
@@ -599,6 +605,21 @@ def build_parser():
                             default='all', help='Determines whether creating manifest file automatically.')
     parser_get.set_defaults(func=get)
 
+    parser_manifest = subparsers.add_parser(
```

Comment:
Additional CLI options to facilitate manifest generation? For example, `--add_prov_cols`.